src/symbol/placement.js

```diff
@@ -956,17 +957,15 @@ export class Placement {
             if (hasIcon) {
                 const packedOpacity = packOpacity(opacityState.icon);
 
-                const useHorizontal = !(hasIconTextFit && symbolInstance.verticalPlacedIconSymbolIndex && horizontalHidden);
-
                 if (symbolInstance.placedIconSymbolIndex >= 0) {
-                    const horizontalOpacity = useHorizontal ? packedOpacity : PACKED_HIDDEN_OPACITY;
```

Comment:
This changed from `!hasIconTextFit || !verticalPlacedIconSymbolIndex || !horizontalHidden` to just `!horizontalHidden`. I understand why `!verticalPlacedIconSymbolIndex` would be dropped but I'm not sure about `!hasIconTextFit`. Do you know what the purpose of `!hasIconTextFit` was in the previous version?